bootstrap/021_external_functions.sql

```diff
@@ -412,7 +372,7 @@ BEGIN
 
             create or replace external function internal.ef_run(unused object, request object)
             returns object
-            context_headers = (CURRENT_ACCOUNT, CURRENT_USER, CURRENT_ROLE, CURRENT_DATABASE, CURRENT_SCHEMA)
+            context_headers = (CURRENT_ACCOUNT, CURRENT_USER, CURRENT_ROLE, CURRENT_DATABASE, CURRENT_SCHEMA, CURRENT_REGION)
```

Comment:
Is this necessary for the `ef_run` endpoint to handle the request or are you just adding this for completeness?

Reply:
yes, this is required for ef_run to work ([here](https://github.com/sundeck-io/sundeck/blob/f23ef6eeaa39234715275e4786094a5a97639ee9/services/query_service/pkg/udf_run_handler.go#L164-L171))